src/init.php

```diff
@@ -455,6 +450,13 @@ public function load_plugin_textdomain() {
 		public function init_animation() {
 			echo '<script>requestAnimationFrame(() => document.body.classList.add( "stk--anim-init" ))</script>';
 		}
+
+		public function init_stackable_vars() {
+			$args = apply_filters( 'stackable_localize_frontend_script', array() );
+			if ( array_key_exists( 'restUrl', $args ) ) {
+				echo '<script> stackable = {"restUrl": "' . $args["restUrl"] .'"} </script>';
```

Comment:
instead of writing the `restUrl` manually, please convert `$args` to JSON instead.